site/en/tutorials/keras/save_and_load.ipynb

```diff
@@ -798,14 +798,14 @@
         "id": "dGXqd4wWJl8O"
       },
       "source": [
-        "Keras saves models by inspecting the architecture. This technique saves everything:\n",
+        "Keras saves models by inspecting their architecture. This technique saves everything:\n",
         "\n",
         "* The weight values\n",
         "* The model's architecture\n",
-        "* The model's training configuration(what you passed to compile)\n",
-        "* The optimizer and its state, if any (this enables you to restart training where you left)\n",
+        "* The model's training configuration (what you pass to the .compile method)\n",
+        "* The optimizer and its state, if any (this enables you to restart training where you left off)\n",
         "\n",
-        "Keras is not able to save the `v1.x` optimizers (from `tf.compat.v1.train`) since they aren't compatible with checkpoints. For v1.x optimizers, you need to re-compile the model after loading—losing the state of the optimizer.\n"
```

Comment:
In my view, the comma should stay here. Reference: [Commas after introductory words and phrases](https://developers.google.com/style/commas?hl=en#commas-after-introductory-words-and-phrases) (Google developer docs style guide).